Makefile

```diff
@@ -1,5 +1,5 @@
 .DEFAULT_GOAL := release
-
+.ONESHELL:
```

Comment:
What does this do? :)

Reply:
Previously each task line runs in a new shell, this allows easier cd into directory and run from there.